src/miner.cpp

```diff
@@ -582,6 +582,24 @@ CBlockTemplate* CreateNewBlockWithKey(CReserveKey& reservekey, CWallet* pwallet,
     if (!reservekey.GetReservedKey(pubkey))
         return NULL;
 
+    const int nHeightNext = chainActive.Tip()->nHeight + 1;
+    static int nLastPOWBlock = Params().LAST_POW_BLOCK();
```

Comment:
Not sure it's useful to redefine the var here instead of calling the function anytime it's needed.

Reply:
I didn't want to go so far and make sure Params() and LAST_POW_BLOCK() would be safe to convert to inline functions; but it didn't seem to make sense to make 4 function calls to pull a constant that wouldn't ever change.   I still have my underlying efficiency eye and hate function calls and division.  Since I put "efficiency" on the PR title, I feel justified :)

Reply:
Sounds reasonable.